### ISSUE TITLE
Update the astropy package-templates and astropy-helpers to their most recent version

### ISSUE THREAD

**Issue by catherine-martlin** [ExoCTK/ExoCTK#98]
I discovered that sphinx can't run to build our readthedocs because our astropy-helpers is very out of date. We want to test out how updating that will or won't break things and eventually see if we can update all of our astropy package dependencies to be the most up to date ones.

**Comment by catherine-martlin** [comment]
See #89, #97, #66

**Comment by hover2pi** [comment]
@catherine-martlin @JeffValenti , we're going to hold off on this until @jaymedina tests the Python version of `exotransmit` called `platon`.

**Comment by catherine-martlin** [comment]
I took today to look more into the sphinx issues and have some updates available on the other open issue (#97 ). It didn't depend much on the `exotransit` package itself and was mostly testing on a branch I'll likely abandon, but I was able to get sphinx to build pages and to look a bit like our current ReadTheDocs finally! I'm going to hold off for a bit on this until we're all able to meet up again and then we can put together a plan of action for seeing how my fixes do or don't mesh with the results of  Jenny's updates.
